W-22513829-authorization-error-eme-dm (#442)
mulesoft/docs-general: modules/ROOT/pages/exp-troubleshoot.adoc

Some features might be in preview or require specific product access.
.. Ask your administrator about feature availability if needed.

> [[cant-authorize-app-error-for-organizations-with-restricted-connected-app-policy]]

Reviewer: consider shortening this anchor

=== "Can't Authorize App" Error for Organizations with Restricted Connected App Policy

If you see a "Can't Authorize App" error when attempting to access the enhanced MuleSoft experience:

. Understand the cause:
+

> This error occurs when your organization's *Connected App Authorization Policy* is set to *restricted*.

Reviewer: policy name not bold in the other file.

+

> .. By default, newly created organizations have this policy set to *unrestricted*, which allows users to access connected apps without additional approval.

Reviewer: This should be a bullet list, not numbered steps.

.. When the policy is set to *restricted*, it indicates that your administrator has intentionally configured this security setting to control which connected apps can access organizational data.
.. This restricted policy takes precedence over any application allowlisting configured on the MuleSoft side.
+

> For more information about Connected App Authorization Policies, see xref:access-management::connected-apps-org-admin.adoc#configure-authorization-policies[Configure Authorization Policies].

Reviewer: consider making this lowercase, since it's not the actual name.

. Contact your administrator:
+

> Only an administrator can resolve this issue. Regular users can't add apps to the allowlist.

Reviewer: Need a better word for "regular"

+

> .. Inform your administrator that the MuleSoft Omni app needs to be added to your organization's allowlist.

Reviewer: bullet list? It doesn't seem sequential

.. Provide them with the error message screenshot if helpful.

. Administrator resolution steps:
+
If you are an administrator:
+
.. When the "Can't Authorize App" error appears, click the *Go to add the app to allowlist* button.
.. Complete the authorization flow to add MuleSoft Omni to your organization's allowlist.

> .. This is a *one-time action* that applies to the entire organization.

Reviewer: ```suggestion
+
This is a one-time action that applies to the entire organization.
```

+

> After completing this step, all users in the organization will be able to access the enhanced MuleSoft experience without encountering this error.

Reviewer: ```suggestion
After completing this step, all users in the organization can access the enhanced MuleSoft experience without encountering this error.
```

. Verify resolution:
+

> If you continue to experience issues after your administrator has added the app to the allowlist:

Reviewer: ```suggestion
If you continue to experience issues after your administrator adds the app to the allowlist:
```

Some features might be in preview or require specific product access.
.. Ask your administrator about feature availability if needed.

[[cant-authorize-app-error-for-organizations-with-restricted-connected-app-policy]]
=== "Can't Authorize App" Error for Organizations with Restricted Connected App Policy

If you see a "Can't Authorize App" error when attempting to access the Enhanced MuleSoft Experience:

. Identify the error:
+
The error message displays:
+
----
Can't Authorize App
Mulesoft Omni - US → [Your Organization Name]

Mulesoft Omni - US wants to access your [username] account in the [Organization Name] organization.

> As an admin, you can add this app to the allowlist for your organization.
> ----
> +
> The dialog includes a button labeled *Go to add the app to allowlist* and a link to *Sign in with Another Account*.
> 
> . Understand the cause:
> +
> This error occurs when your organization's *Connected App Authorization Policy* is set to *restricted*.
> +
> .. By default, newly created organizations have this policy set to *unrestricted*, which allows users to access connected apps without additional approval.
> .. When the policy is set to *restricted*, it indicates that an organization administrator has intentionally configured this security setting to control which connected apps can access organizational data.
> .. This restricted policy takes precedence over any application allowlisting configured on the MuleSoft side.
> +
> For more information about Connected App Authorization Policies, see xref:access-management::connected-apps-org-admin.adoc#configure-authorization-policies[Configure Authorization Policies].
> 
> . Contact your organization administrator:
> +
> Only an *organization administrator* can resolve this issue. Regular users cannot add apps to the allowlist.
> +
> .. Inform your administrator that the MuleSoft Omni app needs to be added to your organization's allowlist.
> .. Provide them with the error message screenshot if helpful.
> 
> . Administrator resolution steps:
> +
> If you are an organization administrator:
> +
> .. When the "Can't Authorize App" error appears, click the *Go to add the app to allowlist* button.
> .. Complete the authorization flow to add MuleSoft Omni to your organization's allowlist.
> .. This is a *one-time action* that applies to the entire organization.
> +
> After completing this step, all users in the organization will be able to access the Enhanced MuleSoft Experience without encountering this error.
> 
> . Verify resolution:
> +
> If you continue to experience issues after your administrator has added the app to the allowlist:
> +
> .. Clear your browser cache and cookies.
> .. Verify that you're signed in with the correct organizational account.
> .. Confirm that your network connectivity to `omni.mulesoft.com` is functioning properly.
> .. Contact your organization's Anypoint Platform administrator for additional assistance.
> 

Author: ```suggestion
[[cant-authorize-app-error-for-organizations-with-restricted-connected-app-policy]]
=== "Can't Authorize App" Error for Organizations with Restricted Connected App Policy

If you see a "Can't Authorize App" error when attempting to access the enhanced MuleSoft experience:

. Identify the error:
+
The error message displays:
+
----
Can't Authorize App
Mulesoft Omni - US → [Your Organization Name]

Mulesoft Omni - US wants to access your [username] account in the [Organization Name] organization.

As an admin, you can add this app to the allowlist for your organization.
----
+
The dialog includes a button labeled *Go to add the app to allowlist* and a link to *Sign in with Another Account*.

. Understand the cause:
+
This error occurs when your organization's *Connected App Authorization Policy* is set to *restricted*.
+
.. By default, newly created organizations have this policy set to *unrestricted*, which allows users to access connected apps without additional approval.
.. When the policy is set to *restricted*, it indicates that your administrator has intentionally configured this security setting to control which connected apps can access organizational data.
.. This restricted policy takes precedence over any application allowlisting configured on the MuleSoft side.
+
For more information about Connected App Authorization Policies, see xref:access-management::connected-apps-org-admin.adoc#configure-authorization-policies[Configure Authorization Policies].

. Contact your administrator:
+
Only an administrator can resolve this issue. Regular users can't add apps to the allowlist.
+
.. Inform your administrator that the MuleSoft Omni app needs to be added to your organization's allowlist.
.. Provide them with the error message screenshot if helpful.

. Administrator resolution steps:
+
If you are an administrator:
+
.. When the "Can't Authorize App" error appears, click the *Go to add the app to allowlist* button.
.. Complete the authorization flow to add MuleSoft Omni to your organization's allowlist.
.. This is a *one-time action* that applies to the entire organization.
+
After completing this step, all users in the organization will be able to access the enhanced MuleSoft experience without encountering this error.

. Verify resolution:
+
If you continue to experience issues after your administrator has added the app to the allowlist:
+
.. Clear your browser cache and cookies.
.. Verify that you're signed in with the correct organizational account.
.. Confirm that your network connectivity to `omni.mulesoft.com` is functioning properly.
.. Contact your organization's Anypoint Platform administrator for additional assistance.

```
